# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/neofetch/hotfix-ia64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- neofetch-7.1.0/neofetch.vanilla	2025-10-04 13:07:10.800055351 +0200
+++ neofetch-7.1.0/neofetch	2025-10-04 13:13:52.196126952 +0200
@@ -2081,11 +2081,17 @@
                     cpu="$(awk -F'=' '/machine/ {print $4; exit}' "$cpu_file")"
                 ;;
 
-                "ia64" | "m32r")
+                "m32r")
                     cpu="$(awk -F':' '/model/ {print $2; exit}' "$cpu_file")"
                     [[ -z "$cpu" ]] && cpu="$(awk -F':' '/family/ {printf $2; exit}' "$cpu_file")"
                 ;;
 
+		 "ia64")
+                    cpu="$(awk -F':' '/model name/ {print $2; exit}' "$cpu_file")"
+                    [[ -z "$cpu" ]] && cpu="$(awk -F':' '/family/ {printf $2; exit}' "$cpu_file")"
+                ;;
+
+
                 *)
                     cpu="$(awk -F '\\s*: | @' \
                             '/model name|Hardware|Processor|^cpu model|chip type|^cpu type/ {
